tools/mass_perldoc_md_creation.pl

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/perl -w
+
+=pod
+
+=head1 NAME
+
+mass_perldoc_md_creation.pl -- Script to mass produce the .md files derived
+from the documentation of the perl scripts and libraries.
+
+=head1 SYNOPSIS
+
+perl mass_perldoc_md_creation.pl C<[options]>
+
+Available options are:
+
+-profile: name of the config file in C<../dicom-archive/.loris_mri>
+
+-verbose: be verbose (boolean)
+
+
+=head1 DESCRIPTION
+
+This script will need to be run once per release to make sure the .md files
```

Comment:
here too for `.md` if you opted for the C<> option above

Reply:
done